### PR TITLE
Add isa-l backend

### PR DESCRIPTION
**Pull request by lamphamsy**
Add isa-l backends

### CODE REVIEW

**Review by rahulreddy — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -10,20 +11,35 @@ ECLIB		= $(dir $(abspath $(MAKEFILE_LIST)))
...
+		&& ./configure --prefix=$(LIBDIR) \
+		LDFLAGS=-L$(LIBDIR)/lib \
+		CPPFLAGS=-I$(LIBDIR)/include
+
+$(ISALLIB): | $(ISALPATH)/Makefile
+	$(MAKE) -C $(ISALPATH) && $(MAKE) -C $(ISALPATH) install
+
 $(GFPATH):
-	git clone http://lab.jerasure.org/jerasure/gf-complete.git $@
+	git clone git@github.com:lamphamsy/gf-complete.git $@
```
Comment (ghost) [inline Makefile]: We can forego this.

File: Makefile [hunk 2]
```diff
@@ -32,7 +48,7 @@ $(GFLIB): | $(GFPATH)/Makefile
 	$(MAKE) -C $(GFPATH) && $(MAKE) -C $(GFPATH) install
 
 $(JERAPATH):
-	git clone http://lab.jerasure.org/jerasure/jerasure.git $@
+	git clone git@github.com:lamphamsy/jerasure.git $@
```
Comment (ghost) [inline Makefile]: As well as this.

File: test/functional/functest.js [hunk 3]
```diff
@@ -167,9 +176,3 @@ describe('FuncTest', function(done) {
         });
     })
 });
-
-// The tests with ISA require an additional library. They are disabled for now because of this.
```
Comment (ghost) [inline test/functional/functest.js:171]: If you added the backend, doesn't it make sense to activate them instead?

In any case, some tests would be welcome here.

**Review by ghost — CHANGES_REQUESTED** [review]
Two nits and a concern.

File: test/functional/functest.js (hunk 3 above)
Comment (lamphamsy) [reply test/functional/functest.js:171]: An example for isa-l is added.

**Comment by lamphamsy** [comment]
Merge to working branch

**Pull request merged**